[UUM-91484] UnityProject example: set theme for unity game activity to make it fullscreen on 6.1

TiphaineRousseauU opened the pull request:
# Description

**Jira link**: [UUM-91484](https://jira.unity3d.com/browse/UUM-91484)

On Unity 6.1, the UnityGameActivity example is broken because of the action bar, which shifts all of the input coordinates by the height of the bar. See attached video on [UUM-91483]( https://jira.unity3d.com/browse/UUM-91483)

This PR forces the UnityGameActivity from the example to be fullscreen even on Unity 6.1. This is done by using the NoActionBar theme in the Android manifest.

TiphaineRousseauU committed:
fix: set theme for unity game activity to make it fullscreen

todi1856 commented on UnityProject/Assets/Plugins/Android/MainApp.androidlib/src/main/AndroidManifest.xml:
```diff
@@ -30,6 +30,7 @@
             android:name=".MainUnityGameActivity"
             android:configChanges="mcc|mnc|locale|touchscreen|keyboard|keyboardHidden|navigation|orientation|screenLayout|uiMode|screenSize|smallestScreenSize|fontScale|layoutDirection|density"
             android:hardwareAccelerated="false"
+            android:theme="@style/AppTheme.NoActionBar"
```
I think it would be better to match Unity themes here. See https://github.cds.internal.unity3d.com/unity/unity/blob/trunk/PlatformDependent/AndroidPlayer/Resources/UnityManifest.xml

Could you set
```
android:theme="@style/UnityThemeSelector"
```
for Activity

and 
```
android:theme="@style/BaseUnityGameActivityTheme"
```
for GameActivity?

TiphaineRousseauU replied:
That makes sense! 
Done

todi1856 reviewed (changes requested):
We probably need to expand our UAAL test here - https://github.cds.internal.unity3d.com/unity/unity/blob/trunk/Tests/Unity.IntegrationTests/Android/AndroidUAAL.cs, by injecting touch event and see that it's being received at the expected position. Since currently we don't validate that, and that's how this bug creeped in.

Anyways, this is optional, I made a memo for myself, unless you want to do it.


Thanks for finding the cause for this issue!

TiphaineRousseauU committed:
review: use unity theme instead

note: android support starts with 2023.1.7f1, for which the used theme is available

TiphaineRousseauU commented:
> We probably need to expand our UAAL test here - https://github.cds.internal.unity3d.com/unity/unity/blob/trunk/Tests/Unity.IntegrationTests/Android/AndroidUAAL.cs, by injecting touch event and see that it's being received at the expected position. Since currently we don't validate that, and that's how this bug creeped in.

Hi (and thanks for the review!) I probably will not have time to do that before the holidays (aka in the next 2h), but I could take a look next year
Do I read [this](https://github.cds.internal.unity3d.com/unity/unity/blob/trunk/Tests/Unity.IntegrationTests/Android/AndroidUAAL.cs#L128) correctly though: the GameActivity test is currently not run, unless you have access to a physical device? Should that be fixed first?

todi1856 commented:
> > We probably need to expand our UAAL test here - https://github.cds.internal.unity3d.com/unity/unity/blob/trunk/Tests/Unity.IntegrationTests/Android/AndroidUAAL.cs, by injecting touch event and see that it's being received at the expected position. Since currently we don't validate that, and that's how this bug creeped in.
> 
> Hi (and thanks for the review!) I probably will not have time to do that before the holidays (aka in the next 2h), but I could take a look next year Do I read [this](https://github.cds.internal.unity3d.com/unity/unity/blob/trunk/Tests/Unity.IntegrationTests/Android/AndroidUAAL.cs#L128) correctly though: the GameActivity test is currently not run, unless you have access to a physical device? Should that be fixed first?

It's up to Lukas to fix - https://jira.unity3d.com/browse/ANT-2623 you can poke him if you want
